homeassistant/components/hunterdouglas_powerview/cover.py

```diff
@@ -299,3 +321,75 @@ def _async_update_shade_from_group(self):
             return
         self._async_process_new_shade_data(self.coordinator.data[self._shade.id])
         self.async_write_ha_state()
+
+
+class SilhouettePowerViewShade(PowerViewShade):
+    """Class to represent a Silhouette (Type 23) blind."""
+
+    def __init__(self, coordinator, device_info, room_name, shade, name):
+        """Initialize the shade."""
+        super().__init__(coordinator, device_info, room_name, shade, name)
+        self._max_tilt = 32767
```

Comment:
Let's make this a constant

Reply:
This should really be exposed by the shade type in `aiopvapi`, but it isnt, i should be able to read it from `shade.max_tilt` or something. I'll make a PR over there too and will come back if it ever gets merged.

Reply:
Are you sure it is 32767?  The api docs seem to is 65535 but they aren't always correct (and it would not surprise me if they are wrong)

PositionValue
The value, with 0 = closed and 65535 = open.

Reply:
Absolutely, the Silhouette blinds only tilt 90 degrees, others tilt the full 180.

so open/65535 = 180 degrees, closed/0 = 0 degrees in HD parlance. As the Silhouettes only tilt "half way", you need to send 32767, anything aboves that upsets them and they need resetting (to my repeated annoyance during debugging)